src/aat/java/uk/gov/hmcts/ccd/userprofile/tests/BaseTest.java

```diff
@@ -12,7 +12,6 @@
 
     protected BaseTest(AATHelper aat) {
         this.aat = aat;
-        RestAssured.baseURI = aat.getTestUrl();
```

Comment:
What's the motivation behind that change?

Reply:
SpotBugs comment says its naughty to alter static fields, hence the change

Reply:
Well, then we may be setting the BaseURI incorrectly. Setting the base URI inside the test works here because we only have a single test, but it will create a fair amount of duplication in the other projects.